File: Data Structures & Algorithms/single-number/submission-0.py
class Solution:
    def singleNumber(self, nums: List[int]) -> int:
        hashmap = {}
        for num in nums:
            hashmap[num] = 0
        for num in nums: 
            hashmap[num] += 1
        for num in hashmap:
            if hashmap[num] == 1:
                return num

File: Data Structures & Algorithms/two-integer-sum/submission-0.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        vector<int> sum;
        int size{nums.size()};
        for (int i{0}; i < size; ++i) {
            for (int j{i+1}; j < size; ++j) {
                if (nums[i] + nums[j] == target) {
                    sum.push_back(i);
                    sum.push_back(j);
                    return sum;
                }
            }
        }
    }
};
